src/lib/y2storage/disk_size.rb

```diff
@@ -38,7 +38,7 @@ class DiskSize
 
     UNITS = ["B", "KiB", "MiB", "GiB", "TiB", "PiB", "EiB", "ZiB", "YiB"]
     # International System units
```

Comment:
np: I would fix this comment to read like `International System of Units (SI)` so the renaming is more clear.